{
  "uid" : "7d4be65dfc205b88",
  "name" : "Validate products API",
  "fullName" : "Schema-Tests.Schemas.Produtos.TestProdutosSchema.Validate products API",
  "historyId" : "c2648335ca9d17176e02a3cb2d9de4d2",
  "time" : {
    "start" : 1655220746241,
    "stop" : 1655220746856,
    "duration" : 615
  },
  "status" : "failed",
  "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7efddfbf84f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
  "flaky" : true,
  "newFailed" : true,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7efddfbf84f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
    "steps" : [ {
      "name" : "ProdutoKW.Search All Products",
      "time" : {
        "start" : 1655220746242,
        "stop" : 1655220746854,
        "duration" : 612
      },
      "status" : "failed",
      "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7efddfbf84f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "steps" : [ {
        "name" : "BuiltIn.Create Dictionary",
        "time" : {
          "start" : 1655220746242,
          "stop" : 1655220746243,
          "duration" : 1
        },
        "status" : "passed",
        "steps" : [ ],
        "attachments" : [ {
          "uid" : "4712b2f619463e4b",
          "name" : "Keyword Log",
          "source" : "4712b2f619463e4b.html",
          "type" : "text/html",
          "size" : 75
        } ],
        "parameters" : [ {
          "name" : "arg1",
          "value" : "content-type=application/json"
        } ],
        "stepsCount" : 0,
        "attachmentsCount" : 1,
        "shouldDisplayMessage" : false,
        "hasContent" : true
      }, {
        "name" : "RequestsLibrary.Create Session",
        "time" : {
          "start" : 1655220746243,
          "stop" : 1655220746244,
          "duration" : 1
        },
        "status" : "passed",
        "steps" : [ ],
        "attachments" : [ {
          "uid" : "a860a30e382b9450",
          "name" : "Keyword Log",
          "source" : "a860a30e382b9450.html",
          "type" : "text/html",
          "size" : 219
        } ],
        "parameters" : [ {
          "name" : "arg1",
          "value" : "serveRest"
        }, {
          "name" : "arg2",
          "value" : "${BASE_URL}"
        }, {
          "name" : "arg3",
          "value" : "verify=True"
        } ],
        "stepsCount" : 0,
        "attachmentsCount" : 1,
        "shouldDisplayMessage" : false,
        "hasContent" : true
      }, {
        "name" : "RequestsLibrary.GET On Session",
        "time" : {
          "start" : 1655220746244,
          "stop" : 1655220746852,
          "duration" : 608
        },
        "status" : "failed",
        "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7efddfbf84f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "steps" : [ ],
        "attachments" : [ {
          "uid" : "31b38aada7b6aee2",
          "name" : "Keyword Log",
          "source" : "31b38aada7b6aee2.html",
          "type" : "text/html",
          "size" : 1228
        } ],
        "parameters" : [ {
          "name" : "arg1",
          "value" : "serveRest"
        }, {
          "name" : "arg2",
          "value" : "/produtos"
        }, {
          "name" : "arg3",
          "value" : "headers=${headers}"
        }, {
          "name" : "arg4",
          "value" : "expected_status=any"
        } ],
        "stepsCount" : 0,
        "attachmentsCount" : 1,
        "shouldDisplayMessage" : true,
        "hasContent" : true
      }, {
        "name" : "BuiltIn.Log",
        "time" : {
          "start" : 1655220746853,
          "stop" : 1655220746853,
          "duration" : 0
        },
        "status" : "failed",
        "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7efddfbf84f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "steps" : [ ],
        "attachments" : [ ],
        "parameters" : [ {
          "name" : "arg1",
          "value" : "${response}"
        } ],
        "stepsCount" : 0,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : true,
        "hasContent" : true
      }, {
        "name" : "BuiltIn.Set Test Variable",
        "time" : {
          "start" : 1655220746853,
          "stop" : 1655220746853,
          "duration" : 0
        },
        "status" : "failed",
        "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7efddfbf84f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "steps" : [ ],
        "attachments" : [ ],
        "parameters" : [ {
          "name" : "arg1",
          "value" : "${response}"
        } ],
        "stepsCount" : 0,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : true,
        "hasContent" : true
      } ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 5,
      "attachmentsCount" : 3,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "BuiltIn.Should Be Equal As Strings",
      "time" : {
        "start" : 1655220746854,
        "stop" : 1655220746855,
        "duration" : 1
      },
      "status" : "failed",
      "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7efddfbf84f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "arg1",
        "value" : "${response.status_code}"
      }, {
        "name" : "arg2",
        "value" : "200"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "JsonValidator.Validate Jsonschema From File",
      "time" : {
        "start" : 1655220746855,
        "stop" : 1655220746855,
        "duration" : 0
      },
      "status" : "failed",
      "statusMessage" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7efddfbf84f0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "arg1",
        "value" : "${response.json()}"
      }, {
        "name" : "arg2",
        "value" : "${PRODUCT_SCHEMA}"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 8,
    "attachmentsCount" : 3,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "Schema-Tests"
  }, {
    "name" : "suite",
    "value" : "Schemas"
  }, {
    "name" : "subSuite",
    "value" : "Produtos.TestProdutosSchema"
  }, {
    "name" : "framework",
    "value" : "robotframework"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "host",
    "value" : "fv-az459-504"
  }, {
    "name" : "thread",
    "value" : "default-40586569-262d-4657-8464-77043a354c14"
  }, {
    "name" : "tag",
    "value" : "@produtos"
  }, {
    "name" : "tag",
    "value" : "@regression"
  }, {
    "name" : "tag",
    "value" : "@schema_tests"
  }, {
    "name" : "tag",
    "value" : "@produtos"
  }, {
    "name" : "tag",
    "value" : "@regression"
  }, {
    "name" : "tag",
    "value" : "@schema_tests"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 3,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 4,
        "unknown" : 0,
        "total" : 7
      },
      "items" : [ {
        "uid" : "761d0b3d998c3bca",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/9//#testresult/761d0b3d998c3bca",
        "status" : "passed",
        "time" : {
          "start" : 1655219658371,
          "stop" : 1655219658452,
          "duration" : 81
        }
      }, {
        "uid" : "caf8c07fa30943cb",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/8//#testresult/caf8c07fa30943cb",
        "status" : "passed",
        "time" : {
          "start" : 1655218426583,
          "stop" : 1655218426697,
          "duration" : 114
        }
      }, {
        "uid" : "694847d0dc1394d2",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/7//#testresult/694847d0dc1394d2",
        "status" : "passed",
        "time" : {
          "start" : 1655217742949,
          "stop" : 1655217743036,
          "duration" : 87
        }
      }, {
        "uid" : "894561fea7704d39",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/5//#testresult/894561fea7704d39",
        "status" : "failed",
        "statusDetails" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos (Caused by ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')))",
        "time" : {
          "start" : 1655217379706,
          "stop" : 1655217380324,
          "duration" : 618
        }
      }, {
        "uid" : "8b7f0614b38badb9",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/4//#testresult/8b7f0614b38badb9",
        "status" : "failed",
        "statusDetails" : "ConnectionError: HTTPConnectionPool(host='localhost', port=3030): Max retries exceeded with url: /produtos (Caused by ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')))",
        "time" : {
          "start" : 1655217085469,
          "stop" : 1655217086095,
          "duration" : 626
        }
      }, {
        "uid" : "d61de3b81dcd7d04",
        "reportUrl" : "https://STC-TEAM-PROJECT.github.io/robot-api-stc/3//#testresult/d61de3b81dcd7d04",
        "status" : "passed",
        "time" : {
          "start" : 1655216675965,
          "stop" : 1655216676215,
          "duration" : 250
        }
      } ]
    },
    "tags" : [ "@schema_tests", "@regression", "@produtos" ]
  },
  "source" : "7d4be65dfc205b88.json",
  "parameterValues" : [ ]
}